src/SegmentedToggle/SegmentedToggle.driver.js

```diff
@@ -0,0 +1,11 @@
+export const segmentedToggleDriverFactory = base => {
+  return {
+    /* fulfilled if element in the DOM */
+    exists: async () => await base.exists(),
+    /* get the actual element */
+    element: async () => await base.getNative(),
+    /* selects child by given number (selection begins with 1) */
+    selectChild: async number =>
+      base.$(`[data-click="segmented-toggle-${number}"]`).click(),
```

Comment:
https://github.com/wix/wix-style-react/pull/2797/files#r252577365

```suggestion
      base.$(`[data-hook="segmented-toggle-${number + 1}"]`).click(),
```